.codeclimate.yml

```diff
@@ -0,0 +1,16 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - javascript
+  eslint:
+    enabled: true
+ratings:
+  paths:
+  - "**.js"
+  - "**.jsx"
```

Comment:
@whabib Do we even have any `.jsx` files? This doesn't seem like something we ever use.